src/icm20689.rs

```diff
@@ -123,7 +123,9 @@ mod tests {
 
     #[test]
     fn test_icm20689_pi_4() {
-        let mut imu = Icm20689Device::builder().build().unwrap();
+        let mut imu = Icm20689Device::builder()
+            .build()
+            .expect("Failed to build ICM20689: {error:?}");
```

Comment:
oh, this should be fixed to `"Failed to build ICM20689"`, as this is _not_ a formatted string, same for the others